DATA/EQUIPMENT/engine_equip.ini

```diff
@@ -3212,6 +3214,7 @@ ids_info = 460858
 ; 
 ; Type: $classLong
 ; Speed: -40 to 120 m/s
+; Thuster bonus speed modifier: -70%

```

Comment:
I think this is a very long text. What about "Thruster Speed Modifier"?

Reply:
The idea behind this was that the thruster speed is engine + thruster bonus speed, and players "could" misread it if it didn't specify.

This "could" also just be me "reading" too much into it. Agreed and good suggestion.